cli/flox-core/Cargo.toml

```diff
@@ -20,3 +20,7 @@ tracing.workspace = true
 tempfile.workspace = true
 time.workspace = true
 thiserror.workspace = true
+
+[features]
+default = ["proc_status"]
```

Comment:
> Copies the implementation from flox-watchdog behind conditional compilation (on by default)

(non-blocking) On by default because both `flox-activations` and `flox-watchdog` (which are the size sensitive binaries) need it, SDK doesn't need it but isn't size sensitive, and we may have another size sensitive crate that doesn't need it in the future?

Reply:
Exactly

Reply:
A consumer that doesn't want it would put this in their Cargo.toml

```
flox-core = { path = ..., default-features = false}
```